src/gui/options_banlist.ui

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>BanListOptions</class>
+ <widget class="QDialog" name="BanListOptions">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>360</width>
+    <height>450</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Dialog</string>
```

Comment:
Oops. Fixed and removed redundant group title. 
![_2016-12-14_21-06-24](https://cloud.githubusercontent.com/assets/14874367/21184977/65abe158-c241-11e6-9186-550cbb548fca.png)